README.md

```diff
@@ -1,38 +1,148 @@
+
+In the longer term, the dev team hope to conquer the world but will settle for the the occasional feeling of self-satisfaction.
+
+### What is the problem that _sqm-autorate_ solves?
 CAKE is an algorithm that manages the buffering of data
 being sent/received by an OpenWrt router so that no more
 data is queued than is necessary,
 minimizing the latency ("bufferbloat")
 and improving the responsiveness of a network.
 
-_sqm-autorate_ is undergoing rapid development, so sometimes the documentation lags behind the latest and greatest.
-We do try to keep up and occasionally succeed!
+CAKE is great!
 
-### Requirements
+However out of the box, CAKE makes the assumption that an ISP connection has a stable speed (bandwidth) and a stable latency.
+This is **not** the case for connections on **cable/DOCIS**, or on **LTE/ and other wireless technologies**.
+The achieveable speed and latency on these connections is sensitive to the number of simultaneous active users (from the ISP point of view) and, for wireless, may be sensitive to weather conditions.
 
-This _sqm-autorate_ program is written primarily for OpenWrt 21.02.
-The current developers are not against extending it for OpenWrt 19.07,
-however it is not the priority as none run 19.07.
-If it runs, that's great.
-If it doesn't run and someone works out why, and how to fix it,
-that's great as well.
-If they supply patches for the good of the project, that's even better!
+To make CAKE work well, there is a compromise and trade off between speed and latency.
+If there is need for fairly constant latency, then speed must be given up.
+If there is a need for higher speed, then latency must be given up.
+The compromise in CAKE is measured by the chosen bandwidth or speed.
+And once a choice is made, it is awkward to adjust in CAKE.
 
-_For Testers, Jan 2022:_ For those people running OpenWrt snapshot builds,
-a patch is required for Lua Lanes.
-Details can be found here:
-[https://github.com/sqm-autorate/sqm-autorate/issues/32#issuecomment-1002584519](https://github.com/sqm-autorate/sqm-autorate/issues/32#issuecomment-1002584519)
+Remember that while network speed is nice, beyond a necessary base speed, latency is what makes a network good for gaming, streaming, video calls, and general use.
+And this is multiplied when several people share a common router.
+Particularly if, for example, one is torrenting large files, while another is gaming, and another is in that career changing conference call.
+
+![Image of Bandwidth Compromise](.readme/Bandwidth-Compromise.png)
+
+Image credit [Lynxthecat](https://github.com/lynxthecat)
+
+### What does _sqm-autorate_ do?
+
+#### In consise terms:
+
+_sqm-autorate_ monitors the current latency and current demand for bandwidth as seen on the router,
+then actively adjusts the CAKE speed to get the best current compromise.
+And it does it independently for both uploads and downloads.
```

Comment:
Probably nitpicking a bit here, but I think `And it does so independently` would sound better

Reply:
agreed